src/api/c/events.hpp

```diff
@@ -0,0 +1,20 @@
+/*******************************************************
+ * Copyright (c) 2014, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+
+#pragma once
+
+#include <Event.hpp>
+#include <backend.hpp>
+#include <af/event.h>
+
+using Event = detail::Event;
```

Comment:
using statements should be avoided in header files.

Reply:
I also think it's clearer to refer to this as `detail::Event` everywhere.